test/src/unit-capi-vfs.cc

```diff
@@ -472,7 +475,7 @@ void VFSFx::check_move(const std::string& path) {
 
 #ifndef _WIN32
 void VFSFx::check_copy(const std::string& path) {
```

Comment:
It looks like this test runs on posix, windows, HDFS, and S3.

The Windows path is correctly skipped with the `ifndef _WIN32`, but this still needs to return-early for S3 and HDFS to avoid pathing to `copy_file`/`copy_dir` since they are unimplemented.

It appears that checking if `supports_s3_ || supports_hdfs_` would be sufficient here.